src/pages/api/enums/SceneNodeTypeValueID.md

```diff
@@ -41,12 +43,28 @@ ___
 
 ___
 
+### <a id="complexShape" name="complexShape"></a> complexShape
+
+• **complexShape** = ``"ComplexShape"``
+
+Type of ComplexShapeNode, representing a complex prepackaged shape that appears as a leaf node in the UI
+
+___
+
 ### <a id="ellipse" name="ellipse"></a> ellipse
 
 • **ellipse** = ``"Ellipse"``
 
 ___
 
+### <a id="gridLayout" name="gridLayout"></a> gridLayout
+
+• **gridLayout** = ``"GridLayout"``
+
+Type of GridLayoutNode represents a grid layout in the scenegraph used to create a layout grid that other content can be placed into
```

Comment:
I think this is just supposed to say "Type of GridLayoutNode" and nothing else – it looks like the sandbox copy of this got its JSDocs out of sync with the upstream original one somehow.

Could you do a quick double-check of everything else in SceneNodeTypeValueID too to make sure none of the other docs are stale also?